src/dos/program_imgmount.cpp

```diff
@@ -387,10 +387,10 @@ void IMGMOUNT::Run(void)
 				WriteOut(MSG_Get("PROGRAM_IMGMOUNT_INVALID_IMAGE"));
 				return;
 			}
-			fseek(diskfile, 0L, SEEK_END);
-			uint32_t fcsize = (uint32_t)(ftell(diskfile) / 512L);
+			cross_fseeko(diskfile, 0L, SEEK_END);
+			uint32_t fcsize = static_cast<uint32_t>(cross_ftello(diskfile) / 512L);
```

Comment:
(use file-size call).

I guess `fcsize` is in blocks? (512-byte/block?) - maybe lift `512L` to a `constexpr auto bytes_per_block`?